### PR TITLE
Handle comments get and post in servlet

### PR DESCRIPTION
**Pull request by jyx11011**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -27,13 +26,22 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> data = new ArrayList<>();
```
Comment (anuragshirolkar) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Let's give the variable a more specific name
```suggestion
  ArrayList<String> comments = new ArrayList<>();
```

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -27,13 +26,22 @@
 @WebServlet("/data")
```
Comment (anuragshirolkar) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: "/comments" would be a better url for this api.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -70,6 +68,12 @@ <h3 class="sec-header">Random Facts</h3>
...
       <div id="fact-container"></div>
+
+      <div id="comments-container"></div>
+      <form action="/comments" method="POST">
+        <textarea name="comment" placeholder="Comment" rows="3" cols="80"></textarea>
+        <button type="submit">Submit</button>
+      </form>
     </div>
   </body>
 </html>
```
Comment (SuyashLakhotia) [inline portfolio/src/main/webapp/index.html]: nit: Missing \n at the EOF.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -53,17 +53,22 @@ function shuffleImagesInGallery() {
...
+  fetch('/comments')
     .then(response => response.json())
     .then(json => {
-      const greetingContainer = document.getElementById('greeting-container');
-      for (var message of json) {
-        const messageElement = document.createElement('p');
-        messageElement.innerText = message;
-        greetingContainer.appendChild(messageElement);
+      const commentsContainer = document.getElementById('comments-container');
+      for (var comment of json) {
```
Comment (SuyashLakhotia) [inline portfolio/src/main/webapp/script.js]: Can this be `for (const comment of json)` instead?

**Review by SuyashLakhotia — CHANGES_REQUESTED** [review]

**Review by SuyashLakhotia — APPROVED** [review]

**Pull request merged**